src/google-classroom-extractor/pyproject.toml

```diff
@@ -30,6 +30,7 @@ black = "^20.8b1"
 rope = "^0.17.0"
```

Comment:
Is rope used? Side note: I looked at the project page in GitHub and does not, as far as I can tell, actually tell you how to use the tool.

Reply:
Removed - never used beyond poking around